src/pages/Homepage/index.tsx

```diff
@@ -4,7 +4,9 @@ import Layout from '../../components/Layout';
 const Homepage = () => {
   return (
     <Layout>
-      <p>Homepage</p>
+      <>
```

Comment:
Is those empty parentheses is necessary?

Reply:
It's a short syntax for React Fragment.

Reply:
> Is those empty parentheses is necessary?

As it was mentioned, styles will be written after functional implementation.